### ISSUE TITLE
GITHUB_REPOSITORY does not do what I expected

### ISSUE THREAD

**Issue by WhiteBlackGoose** [softprops/action-gh-release#137]
Hello. I might have misunderstood the docs, but I expected
```yaml
env:
   GITHUB_REPOSITORY: my_org/my_other_repo
```
to push the release into that repository ; instead, it created the release in the current one.

How to create a release into a specified repository?

**Comment by softprops** [comment]
this is not expected. The feature was contributed by a third party developer. I'll have to tag a closer look

**Comment by WhiteBlackGoose** [comment]
It would be nice to have it more obvious that that feature does. Also, please, add a way to push a release to another repo. For now I will stay on GH CLI because I didn't find the way to do it with your action.

**Comment by softprops** [comment]
I agree. I plan on starting to introduce some stricter guidelines on 3rd party contributions. I have to weigh those against the ability for developers to easily fill in on areas that others have needs where I don’t myself

**Comment by ThePieMonster** [comment]
Any update on this?

**Comment by chenrui333** [comment]
I can't reproduce this as a current  bug on the latest  release. Cross-repo publishing uses the documented  input rather than . If you're still seeing a different current problem, please open a new bug report with a minimal repro or sanitized workflow snippet.
